### ISSUE TITLE
next folder missing in node_modules after installing rnv from npm globally

### ISSUE THREAD

**Issue by matiasmateu** [pavjacko/renative#719]
**Describe the bug**
The next module is not listed in node_modules after a npm installation

**To Reproduce**

1. npm install rnv@0.32.0-alpha.42 -g
2. go to /usr/local/lib/node/rnv/node_modules
3. next is not there

**Expected behavior**
next module should be installed

**Screenshots or copy&paste**
If applicable, copy&paste the output of the command. **Run the command with `-i` to get more debugging informations**

**Desktop (please complete the following information):**

-   OS: macOS BigSur 11.0.1 
-   Node Version 16.8
-   RNV Version 0.32.0-alpha.42

**Additional context**

```
`│ [ error ] Cannot find module 'next'
Require stack:
- /Users/spare30/24i/bootstrap/boot37/node_modules/@expo/next-adapter/build/Server.js
- /Users/spare30/24i/bootstrap/boot37/node_modules/@expo/next-adapter/build/index.js
- /Users/spare30/24i/bootstrap/boot37/node_modules/@rnv/engine-rn-next/dist/adapter.js
- /Users/spare30/24i/bootstrap/boot37/node_modules/@rnv/engine-rn-next/dist/index.js
- /Users/spare30/24i/bootstrap/boot37/node_modules/rnv/dist/core/engineManager/index.js
- /Users/spare30/24i/bootstrap/boot37/node_modules/rnv/dist/core/runtimeManager/index.js
- /Users/spare30/24i/bootstrap/boot37/node_modules/rnv/dist/cli/index.js
- /Users/spare30/24i/bootstrap/boot37/node_modules/rnv/dist/index.js
- /Users/spare30/24i/bootstrap/boot37/node_modules/rnv/bin/index.js
Error: Cannot find module 'next'
Require stack:
- /Users/spare30/24i/bootstrap/boot37/node_modules/@expo/next-adapter/build/Server.js
- /Users/spare30/24i/bootstrap/boot37/node_modules/@expo/next-adapter/build/index.js
- /Users/spare30/24i/bootstrap/boot37/node_modules/@rnv/engine-rn-next/dist/adapter.js
- /Users/spare30/24i/bootstrap/boot37/node_modules/@rnv/engine-rn-next/dist/index.js
- /Users/spare30/24i/bootstrap/boot37/node_modules/rnv/dist/core/engineManager/index.js
- /Users/spare30/24i/bootstrap/boot37/node_modules/rnv/dist/core/runtimeManager/index.js
- /Users/spare30/24i/bootstrap/boot37/node_modules/rnv/dist/cli/index.js
- /Users/spare30/24i/bootstrap/boot37/node_modules/rnv/dist/index.js
- /Users/spare30/24i/bootstrap/boot37/node_modules/rnv/bin/index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:94:18)
    at Object.<anonymous> (/Users/spare30/24i/bootstrap/boot37/node_modules/@expo/next-adapter/src/Server.ts:3:1)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)`

```

**Comment by mihaiblaga89** [comment]
Can you post more info about what command are you running? `rnv` does not have `next` as a dependency so you will never find it there. `engine-rn-next` though has a peerDependency on `next`. Check if your renative.json plugins includes `next: "source:rn"`.

**Comment by mihaiblaga89** [comment]
should be fixed in 0.32.0
